<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>desktop</class>
 <widget class="QMainWindow" name="desktop">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="snake_button">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>70</y>
      <width>461</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string>Snake</string>
    </property>
   </widget>
   <widget class="QPushButton" name="tetris_button">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>190</y>
      <width>461</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string>Tetris</string>
    </property>
   </widget>
   <widget class="QPushButton" name="exit_button">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>310</y>
      <width>461</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
